server/README.md

```diff
@@ -1,6 +1,7 @@
 ## Setup
 
 1. Copy `.env.example` to `.env` and fill in the values
+   1. If you are on ACM @ UCSD Dev Team, fields are available in our google drive for step 1 and 2
```

Comment:
can you undo this change (so there are no changes to the server/ folder), otherwise I think merging your PR might take down both portals while the backend action runs